clang/test/CodeGen/debug-info-cas-friendly-option-test.c

```diff
@@ -0,0 +1,5 @@
+// RUN: %clang -cc1 -debug-info-kind=cas-friendly -triple -x86_64-apple-macosx12.0.0 %s -emit-llvm -o - | FileCheck %s
```

Comment:
I don't think it's necessary / legal to specify a triple here. This test would fail if someone were to configure LLVM with just an aarch64 backend for example.